notebooks/ch11/gradient_basics.md

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
Also for this file, thetas were changed to \hat{\theta} instead of \hat{\theta_0}